README.md

```diff
@@ -27,4 +39,5 @@ yarn build
 ```
 
 Checkout the [deployment documentation](https://v3.nuxtjs.org/docs/deployment).# nuxt3-app
-# nuxt3-app
+
+# Bamboo-nuxt3-app
```

Comment:
@mishschmid Just beautify the readme. Is this clear for someone who wants to clone this project? anything missing? You never know who is reading this!😀

Reply:
Looking good 💄 !

I personally would just remove the "nuxt 3 app" part of it. No need to mention that in the title. You are also not saying welcome to the "Bamboo node prettier graphql vue app" 😄  

So I would just go with Bamboo or Bamboo Bikes or whatever the official name is. But totally up to you, just a personal opinion.

Reply:
Good point!😀